src/parser/nmodl_driver.hpp

```diff
@@ -75,35 +68,49 @@ class NmodlDriver {
     /// enable debug output in the bison parser
     bool trace_parser = false;
 
-    /// pointer to the lexer instance being used
-    NmodlLexer* lexer = nullptr;
-
-    /// pointer to the parser instance being used
-    NmodlParser* parser = nullptr;
-
     /// print messages from lexer/parser
     bool verbose = false;
 
     /// root of the ast
     std::shared_ptr<ast::Program> astRoot = nullptr;
 
+    /// The file library for IMPORT directives
+    FileLibrary library{FileLibrary::default_instance()};
+
+    /// The list of open files, and the location of the request.
+    std::map<std::string, location> open_files;
```

Comment:
nitpick: here we do not need ordering. unordered_map?

Reply:
Fixed in d26e5ef676db7da848eff84ac49ed90efa87538f

Reply:
ehm... you changed defined_var but not open_files